launcher/translation/swedish.ts

```diff
@@ -1241,13 +1240,13 @@ Heroes® of Might and Magic® III HD stöds för närvarande inte!</translation>
     </message>
     <message>
         <location filename="../firstLaunch/firstlaunch_moc.cpp" line="330"/>
-        <source>File cannot opened</source>
+        <source>File cannot be opened</source>
```

Comment:
@XCOM-HUB, thank you for fixing the original mistakes but it won't work doing so. This text must perfectly match the source code so it will just not be translated (and it's also the wrong typo that will appear). It should be fixed by the developers in the source code and then be fixed in all the translation files. Have you done this at other locations?

Reply:
I have only done it here. Can you give me the URL for the other location so I can fix it?

Reply:
@Fabrice-TIERCELIN 

I also made a slight change on row/line 353 (capitalized the first letter 'p' - <source>Please upgrade mod</source>